rootfs/scheduler/__init__.py

```diff
@@ -671,7 +673,7 @@ def _get_port(self, image):
             port = int(list(image_info['Config']['ExposedPorts'].keys())[0].split("/")[0])
         except Exception:
             logger.debug("Failed to find port for Docker image {}, defaulting to 5000".format(image))  # noqa
-            port = 5000
+            port = None
```

Comment:
Are you sure about this change? If so then the debug message above may have to change